lib/cf/applications/src/index.js

```diff
@@ -380,7 +381,8 @@ const reportAppUsage = (cfToken, abacusToken, { failure, success }) => {
   }
 
   errors.missingToken = false;
-  let uri = '/v2/app_usage_events?order-direction=asc&results-per-page=50';
+  let uri = '/v2/app_usage_events?order-direction=asc&results-per-page=' +
```

Comment:
We can use templates: ` /v2/app_usage_events?order-direction=asc&results-per-page=${resultsPerPage}`